packages/react/src/avatar-group/AvatarGroup.tsx

```diff
@@ -29,7 +29,12 @@ export const AvatarGroup = forwardRef<HTMLDivElement, AvatarGroupProps>(
         {overflowChildren.length > 0 && (
           <HoverCard>
             <HoverCardTrigger asChild>
-              <Avatar colorScheme="gray" size={size}>
+              <Avatar
+                bg="neutral.150"
+                color="bg.neutral.inverse"
```

Comment:
let's change the `neutral` avatar to have these styles rather than manually tweaking them from here